src/core/energy_storage/electric_storage.jl

```diff
@@ -261,6 +258,8 @@ struct ElectricStorage <: AbstractElectricStorage
     total_rebate_per_kwh::Real
     charge_efficiency::Float64
```

Comment:
could maybe be worth renaming to ac_dc_charge_efficiency and dc_ac_discharge efficiency? Or if not, at least adding a comment here

Reply:
Yeah that's a good thought. Maybe just ac_charge_efficiency and ac_charge_efficiency though? Cause it's dc_charge_efficiency not dc_dc_charge_efficiency.

Reply:
Oh maybe it's worth specifying in all of the names the from and two units. Like having all of these be dc_dc, ac_dc, dc_ac so that it's clear